src/components/Modal.js

```diff
@@ -0,0 +1,18 @@
+import { ReactComponent as CloseIcon } from "../icons/close.svg";
+import HelpCenter from "./HelpCenter";
+
+const Modal = ({ onClose }) => {
+  return (
+    <div className="modal">
+      <div className="modal-header">
+        <h3 className="heading">Need help?</h3>
```

Comment:
We should be using the WordPress translation function here so this can be translated.